setup.py

```diff
@@ -3,7 +3,9 @@
 
 To create the package for pypi.
 
-1. Change the version in __init__.py, setup.py as well as docs/source/conf.py.
+1. Change the version in __init__.py, setup.py as well as docs/source/conf.py. Remove the master from the links in
+   the new models of the README:
+   (https://huggingface.co/transformers/master/model_doc/ -> https://huggingface.co/transformers/model_doc/)
```

Comment:
Actually it's not only in the README but also in the `index.rst`

Reply:
You have to run make fix-copies after will add.

Reply:
Nice!